src/App.tsx

```diff
@@ -8,11 +9,27 @@ const tabs: Tab[] = [
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Selected tab is&nbsp;Tab 1</h1>
-    <div>{tabs.length}</div>
-  </div>
-);
+const App: React.FC = () => {
+  const [selectedTabId, setSelectedTab] = useState(tabs[0].id);
+
+  return (
+    <div className="App">
+      <h1 className="App__title">
+        Selected tab is&nbsp;
+        {tabs.find(tab => tab.id === selectedTabId)?.title}
+      </h1>
```

Comment:
you can save selected tab in App state and use {`Selected tab is ${selectedTab.title}`} instead find